### ISSUE TITLE
to-do list for release

### ISSUE THREAD

**Issue by luiskuhn** [qbic-pipelines/root-tissue-analysis#25]
- we should probably add the logging of loss values (for training loss vs epoch) during training, perhaps also test/val loss, and test IoU per class, in the remote case that a reviewer asks, it is also helpful to plot these curves to check for signs of overfitting, in case the model needs to be modified/switched and retrained (in training repo)

- I need to figure out how to properly make a pipeline parameter for the monte Carlo dropout T, atm it is hardcoded to 10

- I think Gisela also asked us to remove all the direct nf-core references in the code, I think it is mostly done, we just need to change the module/process names, perhaps switch nf-core for nf-root, we would also check how this ASCII art is generated when the pipeline is run, maybe we can replace it for a cool nf-root one
